docs/sources/alerting/set-up/configure-high-availability/_index.md

```diff
@@ -149,3 +145,48 @@ The following metrics can be used for meta monitoring, exposed by the `/metrics`
    ```
+
+## Verify your high availability setup
+
+When running multiple Grafana instances, all alert rules are evaluated on every instance. This multiple evaluation of alert rules is visible in the [state history](ref:state-history) and provides a straightforward way to verify that your high availability configuration is working correctly.
+
+{{% admonition type="note" %}}
+
+If using a mix of `execute_alerts=false` and `execute_alerts=true` on the HA nodes, since the alert state is not shared amongst the Grafana instances, the instances with `execute_alerts=false` do not show any alert status.
+
+The HA settings (`ha_peers`, etc.) apply only to communication between alertmanagers, synchronizing silences and attempting to avoid duplicate notifications, as described in the introduction.
+
+{{% /admonition %}}
+
+You can also confirm your high availability setup by monitoring Alertmanager metrics exposed by Grafana.
+
+| Metric                                               | Description                                                                                                    |
+| ---------------------------------------------------- | -------------------------------------------------------------------------------------------------------------- |
+| alertmanager_cluster_members                         | Number indicating current number of members in cluster.                                                        |
+| alertmanager_cluster_messages_received_total         | Total number of cluster messages received.                                                                     |
+| alertmanager_cluster_messages_received_size_total    | Total size of cluster messages received.                                                                       |
+| alertmanager_cluster_messages_sent_total             | Total number of cluster messages sent.                                                                         |
+| alertmanager_cluster_messages_sent_size_total        | Total number of cluster messages received.                                                                     |
+| alertmanager_cluster_messages_publish_failures_total | Total number of messages that failed to be published.                                                          |
+| alertmanager_cluster_pings_seconds                   | Histogram of latencies for ping messages.                                                                      |
+| alertmanager_cluster_pings_failures_total            | Total number of failed pings.                                                                                  |
+| alertmanager_peer_position                           | Position the Alertmanager instance believes it's in. The position determines a peer's behavior in the cluster. |
+
+You can confirm the number of Grafana instances in your alerting high availability setup by querying the `alertmanager_cluster_members` and `alertmanager_peer_position` metrics.
```

Comment:
A comment here about what to expect from the `alertmanager_peer_position` metric I think would be helpful, something like:

> Under normal operation, the `alertmanager_peer_position` should be different for Grafana instance; the peers should be incrementally numbered starting at zero, for example, with two instances, one instance should have peer position `0` and the other should have peer position `1`.

Reply:
Yes! Addressed by https://github.com/grafana/grafana/pull/92063#pullrequestreview-2260149953